instageo/apps/viz.py

```diff
@@ -68,7 +68,6 @@ def add_raster_to_plotly_figure(
     # Reproject to EPSG:3857 CRS
     xarr_dataset = xarr_dataset.rio.write_crs(from_crs).rio.reproject("EPSG:3857")
     xarr_dataset = xarr_dataset.where(xarr_dataset <= 1, 0)
-    xarr_dataset = xarr_dataset.where(xarr_dataset > 0.8, 0)
```

Comment:
Why did you remove this code part ?

Reply:
For locust breeding ground visualisation I was increasing the threshold for class 1 so that we see where the model is very confident. This is not required for the cropland classification data